ONSdigital/eq-schema-validator
# Add primary person list collector to schema validation and tests

### Context
As part of the effort to allow for designating a list item as a "primary person", schema changes were introduced that required validating.

This does that, and performs a series of similar tests to those performed on normal list collectors.

### How to test
Clone and run as per v3 branch code, then feed compliant and non-compliant schemas and observe the error messages or lack thereof.
